homeassistant/components/switchbot/binary_sensor.py

```diff
@@ -62,8 +87,8 @@ def __init__(
         super().__init__(coordinator, unique_id, mac, name=switchbot_name)
         self._sensor = binary_sensor
         self._attr_unique_id = f"{unique_id}-{binary_sensor}"
-        self._attr_name = f"{switchbot_name} {binary_sensor.title()}"
         self.entity_description = BINARY_SENSOR_TYPES[binary_sensor]
+        self._attr_name = self.entity_description.name or binary_sensor.title()
```

Comment:
Why do we need a fallback? All the descriptions have a name.

Reply:
cleaned up in f090d9fc8a7bb27d8ff658a98bd7847f12eea4dc

Reply:
cleaned up in f090d9fc8a7bb27d8ff658a98bd7847f12eea4dc